classes/notification/PKPNotificationManager.inc.php

```diff
@@ -348,6 +348,35 @@ public function isVisibleToAllUsers($notificationType, $assocType, $assocId) {
 		return $returner;
 	}
 
+	/**
+	 * Get all subscribable notification types along with names and their setting type values
+	 * @return array
+	 */
+	protected function getNotificationSettingsMap() {
```

Comment:
You use this method statically (`NotificationManager::getNotificationSettingsMap()`) but it's not a static method or class. This will throw a PHP Notice.

Let's make this a `public` method and instantiate the class when calling it: 

```
$notificationManager = new NotificationManager();
$a = $notificationManager->getNotificationSettingsMap();
```